app/models/katello/ping.rb

```diff
@@ -65,14 +70,20 @@ def event_daemon_status(status, result)
 
       def ping_katello_events(result)
         exception_watch(result) do
-          status = Katello::EventMonitor::PollerThread.status(refresh: false)
+          status = Katello::EventDaemon::Runner.cached_status(
+            service_class: Katello::EventMonitor::PollerThread,
+            refresh: false
+          )
           event_daemon_status(status, result)
         end
       end
 
       def ping_candlepin_events(result)
         exception_watch(result) do
-          status = Katello::CandlepinEventListener.status(refresh: false)
+          status = Katello::EventDaemon::Runner.cached_status(
+            service_class: Katello::CandlepinEventListener,
```

Comment:
I'd prefer to see the service name references vs the service class ie `:candlepin_events`

Reply:
How will `Katello::Ping` know the name?  It seemed better to me to leave that to `EventDaemon::Runner` (see my other comment).  but thoughts welcome

Reply:
Just like you do here, but give it the service name vs the service class. Since you're now asking the Runner for the status it's better to ask by the 'friendly name' rather than the class which implements the service - that's now an implementation detail we can hide through the Runner

Reply:
but that will cause a lot of difficulty in Monitor and other places where we do `@service_classes.each` loops

Reply:
I think it pretty much comes down to making [this change](https://github.com/Katello/katello/pull/9121#discussion_r566278311) and the rest should fall into place

Reply:
@jturel ok, updated!